commit 7bc79645d4e404d00fee43b5a6cee4e3b6bf7a35
Author: hasufell <hasufell@gentoo.org>
Date:   Sat Sep 7 12:48:58 2013 +0200

    BUILD: fix withXaw conditional
    
    Listing both one after another leads to withXaw being false
    if you pass "--with-Xaw3d --without-Xaw".

--- a/configure.ac
+++ b/configure.ac
@@ -248,8 +248,7 @@ fi
 
 dnl | make results available in Makefile.am
 AM_CONDITIONAL([withGTK],   [test x"$with_GTK" = x"yes"])
-AM_CONDITIONAL([withXaw],   [test x"$with_Xaw3d" = x"yes"])
-AM_CONDITIONAL([withXaw],   [test x"$with_Xaw" = x"yes"])
+AM_CONDITIONAL([withXaw],   [test x"$with_Xaw3d" = x"yes" || test x"$with_Xaw" = x"yes"])
 
 
 AC_SUBST(FRONTEND_CFLAGS)
